build.clj

```diff
@@ -11,25 +12,30 @@
 
 (def source-uri "https://github.com/fluree/db")
 
+(defn compile [_]
```

Comment:
We should add a note to the readme that you need to run `clj -T:build compile` the first time you check out the repository or any time the `fluree.db.SID` class changes.